src/robot_dart/gui/magnum/gs/camera.cpp

```diff
@@ -69,6 +69,14 @@ namespace robot_dart {
 #endif
                 }
 
+                Magnum::Matrix3 intrinsicMatrix()
+                {
+                    Magnum::Matrix4 perspProj = Magnum::Matrix4::perspectiveProjection(_fov, _aspect_ratio, _near_plane, _far_plane);
+                    return {{perspProj[0][0], 0., 0.},
```

Comment:
Not sure this initialization is correct. `Matrix3` accepts 3 column vectors (one for each column). Also the `[]` operator first access columns, then rows..